apps/authentication-portal/src/main/webapp/identifierauth.jsp

```diff
@@ -131,7 +131,7 @@
     %>
     <div class="field">
         <div class="g-recaptcha"
-             data-sitekey="<%=Encode.forHtmlContent(request.getParameter("reCaptchaKey"))%>">
+             data-sitekey="<%=Encode.forHtmlContent(reCaptchaKey)%>">
```

Comment:
Where is `reCaptchaKey` defined?

Reply:
reCaptchaKey is defined in login.jsp and it imports the identifier.jsp